src/Renderer/PointsMaterial.js

```diff
@@ -13,8 +13,7 @@ class PointsMaterial extends RawShaderMaterial {
         this.uniforms.resolution = new Uniform(new Vector2(window.innerWidth, window.innerHeight));
         this.uniforms.pickingMode = new Uniform(false);
         this.uniforms.opacity = new Uniform(1.0);
-        this.uniforms.useCustomColor = new Uniform(false);
-        this.uniforms.customColor = new Uniform(new Color());
+        this.uniforms.rgba = new Uniform(new Vector4(0, 0, 0, 0));
```

Comment:
nitpicking the name: `customColor` was fine. Or maybe `overlayColor`? We need something more precise than rgba here.

Reply:
agreed. rgba is as bad as customColor...
overlayColor sounds good to me.